src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/BulkPrintService.java

```diff
@@ -63,17 +60,15 @@ public BulkPrintService(GenericDocumentService genericDocumentService,
         this.coverSheetService = coverSheetService;
     }
 
-    public UUID sendNotificationLetterForBulkPrint(final CaseDocument notificationLetter, final CaseDetails caseDetails) {
-        List<BulkPrintDocument> notificationLetterList = Collections.singletonList(
-            documentHelper.getCaseDocumentAsBulkPrintDocument(notificationLetter));
+    public UUID sendDocumentForPrint(final CaseDocument document, CaseDetails caseDetails) {
+        List<BulkPrintDocument> bulkPrintDocument = Collections.singletonList(
+            BulkPrintDocument.builder().binaryFileUrl(document.getDocumentBinaryUrl()).build());
 
-        Long caseId = caseDetails.getId();
-        log.info("Notification letter sent to Bulk Print: {} for Case ID: {}", notificationLetterList, caseId);
-
-        return bulkPrintFinancialRemedyLetterPack(caseId, notificationLetterList);
+        return bulkPrintDocuments(caseDetails.getId(), FINANCIAL_REMEDY_GENERAL_LETTER, bulkPrintDocument);
     }
 
-    public UUID sendOrderForBulkPrintRespondent(final CaseDocument coverSheet, final CaseDetails caseDetails) {
+
+    public UUID sendConsentOrderForBulkPrintRespondent(final CaseDocument coverSheet, final CaseDetails caseDetails) {
```

Comment:
can this method be made generic yet rather than consent order?

Reply:
Yep, I agree this would be a good change and should be done but i think i'm out of time on this ticket. 
I only renamed this method didn't change it so i'd be keen to get this ticket merged in to make the other new methods on bulkservice avaliable to everyone else and in the next bulk ticket i print off look at refactoring out the specific stuff in bulkprintservice to leave it a generic service